components/editor/modules/figure/caption.js

```diff
@@ -1,12 +1,14 @@
 import MarkdownSerializer from 'slate-mdast-serializer'
 import { Block } from 'slate'
 import { matchBlock } from '../../utils'
-import Placeholder from '../../Placeholder'
+import { staticKeyHandler } from '../../utils/keyHandlers'
+import { Inline } from '../../Placeholder'
 
 const getSerializer = options => {
+  const [ byLineModule, ...subModules ] = options.subModules
```

Comment:
`byline` is one word and doesn't necessarily need to be camelCased `byLine`, but I leave that up to you:
https://en.wikipedia.org/wiki/Byline